### PR TITLE
Fix : Added a media query for dark mode users

### PR DESCRIPTION
**Pull request by YashNuhash**
**Fixes** https://github.com/expressjs/expressjs.com/issues/1382

### CODE REVIEW

**Comment by YashNuhash** [comment]
Hey, @dougwilson

Now, you can review my commit

File: css/style.css [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 body {
   font: 400 14px/1.6 "Open Sans", sans-serif;
-  background: url(/images/bg.jpg);
+  background: url(/images/bg.jpg);
```
Comment (YashNuhash) [inline css/style.css:17]: Added background

File: css/style.css [hunk 2]
```diff
@@ -669,6 +669,12 @@ footer {
 
 }
 
+@media (prefers-color-scheme:dark) {
```
Comment (YashNuhash) [inline css/style.css:672]: Added media query

**Review by YashNuhash — COMMENTED** [review]
This is all about the changes that I made here.

File: css/style.css [hunk 3]
```diff
@@ -669,6 +669,12 @@ footer {
 
 }
 
+@media (prefers-color-scheme:dark) {
+    .en-doc{
+        background: black;
+    }
+  }
+
```
Comment (YashNuhash) [inline css/style.css:677]: Added media query

File: css/style.css [hunk 4]
```diff
@@ -669,6 +669,12 @@ footer {
 
 }
```
Comment (YashNuhash) [inline css/style.css:671]: Added media query

File: css/style.css (hunk 2 above)
Comment (YashNuhash) [inline css/style.css:672]: Added media query

**Comment by crandmck** [comment]
@chrisdel101 Should this be closed or merged?

**Comment by chrisdel101** [comment]
This would have handled the system setting,  but only changed the main background. So `closed` since #1490 handles this a bit differently. 
(ps - reopen if I closed prematurely :) )

**Pull request closed without merging**